lib/grpcServer/handlers/core/getBlockHandlerFactory.js

```diff
@@ -0,0 +1,54 @@
+const {
+  GetBlockResponse,
+} = require('@dashevo/dapi-grpc');
+
+const { Block } = require('@dashevo/dashcore-lib');
+
+const {
+  server: {
+    error: {
+      InvalidArgumentGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+/**
+ * @param {InsightAPI} insightAPI
+ * @returns {getBlockHandler}
+ */
+function getBlockHandlerFactory(insightAPI) {
+  /**
+   * @typedef getBlockHandler
+   * @param {Object} call
+   * @return {Promise<GetBlockResponse>}
+   */
+  async function getBlockHandler(call) {
+    const { request } = call;
+
+    const height = request.getHeight();
+    const hash = request.getHash();
+
+    if (!hash && !height) {
+      throw new InvalidArgumentGrpcError('hash or height is not specified');
+    }
+
+    let serializedBlock;
+
+    if (hash) {
+      serializedBlock = await insightAPI.getRawBlockByHash(hash);
```

Comment:
it seems it might throw an error if the response contains `error` field https://github.com/dashevo/dapi/pull/210/files#diff-1b0562ab6c26680d91304ad29d6b206bR21. I guess we need to handle invalid param errors from insight.

Reply:
But you told previously, that we don't need it, because we already have internal errors wrapper.

Reply:
Let's respond with an internal error as previously and handle errors properly later.